crates/oxc_query/src/schema.graphql

```diff
@@ -306,6 +306,18 @@ type ObjectLiteral implements Expression & HasSpan {
   span: Span!
 }
 
+type ObjectLiteralAST implements Expression & HasSpan & ASTNode {
+  value(key: String!): [Expression!]!
+
+  # Expression
+  as_constant_string: String
+  # ASTNode
+  parent: ASTNode
+  ancestor: [ASTNode!]!
+  # HasSpan
```

Comment:
maybe provide a more useful message? Maybe even a doc comment?

Reply:
yes, but in a future pr